wandb/sdk/wandb_run.py

```diff
@@ -2134,6 +2134,27 @@ def _swap_artifact_name(self, artifact_name: str, use_as: Optional[str]) -> str:
     def _detach(self) -> None:
         pass
 
+    def link_artifact(
+        self,
+        artifact: Union[public.Artifact, Artifact],
+        portfolio_path: str,
```

Comment:
portfolio or registry ... the world may never know!

Reply:
target for now, I suppose, to be generic :)